lib/main.js

```diff
@@ -120,6 +120,7 @@ module.exports = function (public_key, private_key, options) {
                 var form = new FormData();
                 form.append( 'UPLOADCARE_PUB_KEY', public_key );
                 form.append( 'file', fileStream );
+                form.append( 'store', 'auto' );
```

Comment:
you need to change this to `UPLOADCARE_STORE`: https://uploadcare.com/documentation/upload/#upload-body